4.3/examples

```diff
@@ -0,0 +1,85 @@
+### Configuration Example
+
+A setup example with almost all flag possibilities including a short explanation:
+
+```sh
+
+docker run -it \
+
+	-v nominatim-flatnode:/nominatim/flatnode \
+	#Sets the flatnode file, which is to reduce the load on the database when you plan to use multiple countrys together bigger than 6GB
```

Comment:
Please move the comments above the line they refer to.

Reply:
Done